docs/getting-started-guides/fedora/fedora_ansible_config.md

```diff
@@ -209,7 +209,7 @@ EOF
 kubectl get pods
 ```
 
-Important : Note that the ip of the pods IP fields are on the network which you created in the kube_ip_addr file.
+Important : Note that the IP of the pods IP fields are on the network which you created in the kube_ip_addr file.
```

Comment:
"Note that pods will be assigned IPs from the network which you created in the kube_ip_addr file."

Is this what that sentence is trying to say? I'm not sure what an "IP of the pods IP fields" would be...